plate_simulation/models/params.py

```diff
@@ -69,11 +69,6 @@ class PlateParams(BaseModel):
     reference_surface: str = "topography"
     reference_type: str = "mean"
 
-    @field_validator("plate", mode="before")
```

Comment:
Where are you doing the conversion to account for removing these?  If they're resistivities in the ui.json, and remain resistivities in the params classes, then the driver needs to flip them, but I don't see that change.

Reply:
Happens in series on the whole model.